contrib/epee/tests/src/tests.cpp

```diff
@@ -1,59 +0,0 @@
-
-#include "include_base_utils.h"
-#include "storages/storage_tests.h"
-#include "misc/test_math.h"
-#include "storages/portable_storages_test.h"
-#include "net/test_net.h"
-
-using namespace epee;
-
-int main(int argc, char* argv[])
-{
-
-  string_tools::set_module_name_and_folder(argv[0]);
-
-  //set up logging options
-  log_space::get_set_log_detalisation_level(true, LOG_LEVEL_2);
-  log_space::log_singletone::add_logger(LOGGER_CONSOLE, NULL, NULL);
-  log_space::log_singletone::add_logger(LOGGER_FILE, 
-    log_space::log_singletone::get_default_log_file().c_str(), 
-    log_space::log_singletone::get_default_log_folder().c_str());
-
-
-  string_tools::command_line_params_a start_params;
-  string_tools::parse_commandline(start_params, argc, argv);
-  std::string tests_data_path;
-  string_tools::get_xparam_from_command_line(start_params, std::string("/tests_folder"), tests_data_path);
-
-  if(string_tools::have_in_command_line(start_params, std::string("/run_net_tests")))
```

Comment:
Hmmm. Are these tests already duplicated in Monero's `tests` dir?

Reply:
Good point, we might be able to increase coverage with these tests. Either way, they don't compile, but maybe I can see if I can add these tests to the main tests/ directory.

Reply:
@mj-xmr Here's my more detailed analysis of epee/tests:

epee/tests/src/misc/test_math.h is still valid, will incorporate those these
epee/tests/src/storages/storages_tests.h tests StorageNamed, which doesn't exist anymore
epee/tests/src/storages/portable_storages_test.h uses an outdated version of EPEE serialization
contrib/epee/tests/src/net/test_net.h tests levin_protocol_handler_async.h for which there is already a unit_test
tests.cpp ties them all together